### DISCUSSION TITLE
Is there any way to spawn in creatures?

### DISCUSSION THREAD

**Discussion by ManuelSvQ** [winstxnhdw/lc-hax#119] [Q&A] [answered]
I've been able to spawn a bunker spider outside once but I don't know if it's possible to spawn other things.

**Comment by ManuelSvQ** [comment]
Question 2: It appears whenever I use the infinite scan sometimes it decides to start reloading the screen very quickly, leaving me at 2 FPS and flashing me constantly

**Comment by winstxnhdw** [comment]
We don't support host-only features like spawning mobs. 

Your FPS problem is unusual, and I can't help you with that. Infinite scan simply uses the game's original scan without its limits. Nothing special about the code.

**Comment by winstxnhdw** [comment, marked as answer]
Holy fuck, I've found a way to infinite spawn masked enemies.

**Reply by Isuckwithusernames** [reply]
i need it

**Reply by winstxnhdw** [reply]
It's already added.

**Reply by yujispin990** [reply]
you mean the /mask?

**Reply by winstxnhdw** [reply]
Yeap.

**Reply by winstxnhdw** [reply]
See #131 for caveats, though.

**Reply by yujispin990** [reply]
ah,only 1 mask is attacking and the rest are just idling?

**Reply by winstxnhdw** [reply]
I am not sure. When I spawn many, none of them can attack.
